fink_science/kilonova/processor.py

```diff
@@ -43,7 +43,7 @@ def knscore(jd, fid, magpsf, sigmapsf, model_path=None, pcs_path=None, npcs=None
         Magnitude from PSF-fit photometry, and 1-sigma error
     model_path: Spark DataFrame Column, optional
         Path to the trained model. Default is None, in which case the default
-        model `data/models/model_1PC_2KN_Cosmins_models.pkl` is loaded.
+        model `data/models/KN_model_2PC.pkl` is loaded.
```

Comment:
Why the two models do not have the same name (except the number of PC)? Could you homogenize them (one way or the other) :-)